_includes/field-description-logourl.md

```diff
@@ -0,0 +1,28 @@
+{%- assign documentation_section = include.documentation_section -%}
+{%- capture logo_url-%}
+   {%- if documentation_section == "payment-menu" or documentation_section == "checkout" -%}
+        /{{ documentation_section }}/other-features#custom-logo
+    {%- endif -%}
```

Comment:
Should `logo_url` be blank if `documentation_section` is not `payment-menu` or `checkout`?

Reply:
What about seamless view?

Reply:
If its Payment menu or Checkout we have the following; 
 With permission and activation on your contract, sending in a `logoUrl` will
    replace the Swedbank Pay logo with the logo sent in. If you do not send in a
    `logoUrl`, then no logo and no text is shown. Without permission or activation
    on your contract, sending in a `logoUrl` has no effect. Read more about this in
    [Custom Logo]({{logo_url}}). 

Otherwise, for all other payments we have ;
The URL that will be used for showing the customer logo. Must be a picture 
    with maximum 50px height and 400px width. Require https. 

Its not blank. But When i created the Custom logo for Payment menu, the task only said custom logo in payment menu, not in any other payment instrument. Let me know if this should be in all payment instruments

Reply:
Since there is no `else` clause, `logo_url` will become empty if `documentation_section` is not `payment-menu` or `checkout`. But I see now that `logo_url` is only used in `logo_description` if `documentation_section` is `payment-menu` or `checkout`, so it would be better to just move the construction of `logo_url` inside the `logo_description`.